src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -232,7 +232,7 @@ await requestExecutor.ExecuteAsync(_redirectNode, null, context, command, should
                     TcpConnectionHeaderMessage.OperationTypes.Subscription,
                     NegotiateProtocolVersionForSubscriptionAsync,
                     context,
-                    requestExecutor.DefaultTimeout,
+                    TimeSpan.FromSeconds(30),
```

Comment:
Should be something else?

Reply:
let's have a static var in the subscription worker

Reply:
Let's avoid hard coding it. We should make it user settable

Reply:
settable per worker yes?

Reply:
also what should be the default timeout

Reply:
We can add it to the SubscriptionWorkerOptions, it does not by default need to be super low, IMO we could stick to 30s. @ayende